components/UI/Button.js

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+
+const Button = styled.button`
```

Comment:
Looking at everything below: Is this supposed to be a component not actually consumed inside pages, but only act as a "base component" for all the other buttons? (`ButtonPrimary`, `ButtonInline`, ...)
If so, I'd suggest we
- create `/components/UI/buttons/`,
- move this component there, call the component `BaseButton` and call the file `_base.js`,
- move all other buttons also into that directory (and make them all extend that "BaseButton" instead of just "Button")

All of these points are to highlight that the component itself is most likely not to be used or consumed anywhere -- and that someone would most likely want to use -one- of the derivatives instead.

Reply:
Hm, yes, we could re-structure the files and re-factor to components this way.

Reply:
This would've been my suggestion as well, like they do it in this example: https://styled-components.com/docs/basics#extending-styles 
I would still prefer the naming scheme how it is atm. So `<Button />` is our base button which all other variants extend on. I can also create a PR for this if you want @andirueckel